hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/commit/SparkDeletePartitionCommitActionExecutor.java

```diff
@@ -79,12 +77,7 @@ public HoodieWriteMetadata<HoodieData<WriteStatus>> execute() {
       if (!table.getStorage().exists(
           new StoragePath(table.getMetaClient().getTimelinePath(),
               instantFileNameGenerator.getFileName(dropPartitionsInstant)))) {
-        HoodieRequestedReplaceMetadata requestedReplaceMetadata =
-            HoodieRequestedReplaceMetadata.newBuilder()
-                .setOperationType(WriteOperationType.DELETE_PARTITION.name())
-                .setExtraMetadata(extraMetadata.orElse(Collections.emptyMap()))
-                .build();
-        table.getMetaClient().getActiveTimeline().saveToPendingReplaceCommit(dropPartitionsInstant, requestedReplaceMetadata);
+        throw new HoodieDeletePartitionException("Commit instant was not created for instant " + instantTime);
```

Comment:
This is not right, the requested instant was created with empty `HoodieRequestedReplaceMetadata`,